gsoc/students-guideline.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "HSF GSoC Student Guidelines"
+author: "Javier Cervantes"
+layout: default
+---
+
+# Guideline for Students
+
+### First steps
+
+Get started checking out the [Google Summer of Code Guides for students](https://google.github.io/gsocguides/student/) and the [timeline](https://developers.google.com/open-source/gsoc/timeline) of the program.
+
+### How to apply to CERN-HSF
+
+1. Take a look at the [list of projects](/activities/gsoc.html#projects-in-2019) participating for this year.
+  * There you will find a list of proposals for each project, their description and the contact information of their mentors.
+  * There is also a list with [all the project proposals](http://hepsoftwarefoundation.org/gsoc/2019/summary.html) sorted in alphabetical order.
+
+2. Once you have chosen one or more proposals that you like, please contact the corresponding mentors and they will guide you through the next steps.
+
```

Comment:
Formatting looks a bit odd, with more space between bullets 2 and 3 than between 1 and 2. Maybe try removing the blank lines?

Reply:
I agree, rather than removing blank lines, I have tried to add the same extra space in between. What do you think?

Reply:
Yes, looks better now